src/main/java/com/templateproject/api/entity/ToDoList.java

```diff
@@ -0,0 +1,67 @@
+package com.templateproject.api.entity;
+
+import java.util.List;
+
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+
+@Entity
+public class ToDoList {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
+    @Column(columnDefinition = "boolean default false")
+    private boolean isFavorite;
+
+    @OneToMany(mappedBy = "toDoList", cascade = CascadeType.ALL)
+    private List<Task> tasks;
+
+    public ToDoList() {
```

Comment:
Here a Task will at least have a title, create a constructor with title (and any other necessaries parameters to construct or hydrate the object)